style/style.css

```diff
@@ -0,0 +1,312 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+body{
+    background-color: #ffffff;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: 200px auto auto;
+    grid-template-areas: 
+    "hd"
+    "section"
+    "footer";
+    gap: 1rem;
+}
+
+
+.header{
+    /* background-color: red; */
+    grid-area: hd;
+    background-image: url("/img/header\ pic.avif");
+    object-fit: cover;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    width: 100%;
+}
+
+.nav-header {
+    display: none;
+}
+
+.add {
+    max-width: 24rem;
+    word-wrap: break-word;
+
+}
+
+.add-text {
+    color: rgb(255, 255, 255);
```

Comment:
Putting rgb colour values inline like this works fine 👍
Even better is to use CSS variables to store the colours, will make them readable as you use them in your CSS. You can declare them once in a :root selector then re-use them ([example](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#using_the_root_pseudo-class))